<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ameet" failures="1" tests="11" name="Test" time="76.965" errors="0" timestamp="2025-07-11T11:59:22 IST">
  <testcase classname="testcases.RestAssured_TestCases" name="loginWithSeleniumAndGetCookie" time="34.257"/>
  <testcase classname="testcases.RestAssured_TestCases" name="getSubunitsTree" time="18.396"/>
  <testcase classname="testcases.RestAssured_TestCases" name="getAdminSkills" time="1.23"/>
  <testcase classname="testcases.RestAssured_TestCases" name="getAdminEdu" time="1.152"/>
  <testcase classname="testcases.RestAssured_TestCases" name="getAdminLicenses" time="1.027"/>
  <testcase classname="testcases.RestAssured_TestCases" name="getAdminLanguages" time="0.981"/>
  <testcase classname="testcases.RestAssured_TestCases" name="postNewJobTitles" time="4.522"/>
  <testcase classname="testcases.RestAssured_TestCases" name="putJobTitleByID" time="1.309"/>
  <testcase classname="testcases.RestAssured_TestCases" name="postUniquePaygrade" time="1.236"/>
  <testcase classname="testcases.RestAssured_TestCases" name="deleteValidUsers" time="2.572">
    <failure type="java.lang.AssertionError" message="Expected status code 200 for success. expected [200] but found [422]">
      <![CDATA[java.lang.AssertionError: Expected status code 200 for success. expected [200] but found [422]
at testcases.RestAssured_TestCases.deleteValidUsers(RestAssured_TestCases.java:309)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- deleteValidUsers -->
  <testcase classname="testcases.RestAssured_TestCases" name="deletePaygradeById" time="1.189"/>
</testsuite> <!-- Test -->
